venus/src/pages/spot/components/tag/SpotTag.tsx

```diff
@@ -1,11 +1,14 @@
 type SpotTagProps = {
-    name: string;
+  name: string;
+  textSizeClass?: string;
 };
 
-export default function SpotTag({ name }: SpotTagProps) {
-    return (
-        <div className="border-darkText w-fit rounded-lg border px-3.5 py-1 text-center text-lg">
-            {name}
-        </div>
-    );
+export default function SpotTag({ name, textSizeClass }: SpotTagProps) {
+  return (
+    <div
+      className={`border-darkText w-fit rounded-lg border px-3.5 py-1 text-center ${textSizeClass ? textSizeClass : "text-lg"}`}
```

Comment:
```suggestion
      className={`border-darkText w-fit rounded-lg border px-3.5 py-1 text-center ${textSizeClass || "text-lg"}`}
```
To można skrócić do takiej postaci

Reply:
Zmieniłem